server/utils/checkYourAnswersUtils.test.ts

```diff
@@ -517,4 +519,81 @@ describe('getPage', () => {
       getPage('confirm-eligibility', 'bar')
     }).toThrow(UnknownPageError)
   })
+
+  describe('getApplicantDetails', () => {
```

Comment:
Now we have some test coverage I was thinking we could add a new shorter test for checking the default behaviour

```
person = personFactory.build({nationality: nil})
expected = {
          key: {
            text: 'Nationality',
          },
          value: {
            html: "Unknown",
          },
        },
 expect(getApplicantDetails(application)).toContain(expected)
```

Reply:
we could, but how much value is there in that test? Are we not at that point basically testing the javascript `||` operator?

Reply:
The implementation is simple in this case but so is the test and having a test would protect it from regression and document the behaviour.

Reply:
OK yeah, good points. I have done this 👍